basilisk/src/autotest.sh

```
@@ -13,27 +14,36 @@ EOF
+			while test `ssh $SANDBOX tsp | grep -E '(running|queued)' | wc -l` -ge 10; do
+			    sleep 60;
+			done
 			echo running $d/$f
 			if test -f $basename/pass -a ! -f $basename/fail; then
 			    rm -r -f $basename-pass
 			    cp -ar $basename $basename-pass
 			fi
 			make -f $makefile $f.html $basename.tst
+			sleep 5
 		    fi
 		    cd $HOME/wiki
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)